internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go

```diff
@@ -0,0 +1,24 @@
+package tocamelcasewithadditionalinitialisms
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenCodeHasCorrectNames(t *testing.T) {
+	pet := &Pet{}
+	assert.Equal(t, "", pet.NAME)
+	assert.Equal(t, "", pet.UUID)
+
+	uri := "https://my-api.com/some-base-url/v1/"
+	client, err := NewClient(uri)
+	assert.Nil(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)
```

Reply:
For record, I copy/paste exactly from the other tests ... shall we be consistent?